pkgs/applications/audio/ams/default.nix

```diff
@@ -0,0 +1,39 @@
+{ stdenv
+, fetchgit
+, automake
+, alsaLib
+, ladspaH
+, libjack2
+, fftw
+, zita-alsa-pcmi
+, qt4
+, pkg-config
+, autoreconfHook
+}:
+
+stdenv.mkDerivation rec {
+  name = "ams";
+  version = "2.1.2";
+
+  src = fetchgit {
+    url = https://git.code.sf.net/p/alsamodular/ams.git;
+    sha256 = "0v5lasrgghcjw47dfxzpxp93mxykimh3raz8kpzk2i7z7cnlp5wa";
+    rev = "a47d45272a1c387c1cfc37b9e82d6245429a3117";
```

Comment:
This does not match the commit tagged as `Release-212`: https://sourceforge.net/p/alsamodular/ams.git/ci/0fc54f81a87897fec5370bf6411a9f4ff7bac1a6/

You should use the tag name when available to prevent such mismatches.